index.js

```diff
@@ -28,13 +29,15 @@ module.exports = (oppiabot) => {
     if (whitelistedAccounts.includes(context.repo().owner.toLowerCase())) {
       await apiForSheetsModule.checkClaStatus(context);
       await checkPullRequestLabelsModule.checkChangelogLabel(context);
+      await checkPullRequestBranchModule.checkBranch(context);
       await checkWIPModule.checkWIP(context);
     }
   });
 
   oppiabot.on('pull_request.reopened', async context => {
     if (whitelistedAccounts.includes(context.repo().owner.toLowerCase())) {
       await checkPullRequestLabelsModule.checkChangelogLabel(context);
+      await checkPullRequestBranchModule.checkBranch(context);

```

Comment:
Branch won't be changed when a PR is re-opened -- so why do we need this test here? (One use case is only for the first time when this check is added and a PR is opened & closed before addition of test and reopened after addition of test -- but I don't think that is very important since it will only be useful for initial run).

Reply:
But the PR author might try to reopen the PR after oppiabot closed it.

Reply:
Add a comment for the same.

Reply:
Done.